### ISSUE TITLE
add support for other software under categories

### ISSUE THREAD

**Issue by Dawsson** [flytegg/mcu-website#131]
Plugins: Spigot, Paper, Purpur, Vanilla
Modded: Fabric, Forge, Arclight, Magma, Mohist
Proxy: Bungee, Waterfall, Velocity
Other: Minestom, AdvancedSlimePaper

not all are needed, just some ideas.

**Comment by stephendotgg** [comment]
Would you mind ticking the ones off we have done?
Done: Purpur/Paper/Spigot/Bukkit/Vanilla

Next up will probs be the proxy ones.

**Comment by stephendotgg** [comment]
Added Waterfall, Velocity & Folia 
Trying to figure out Bungee versioning.......

**Comment by stephendotgg** [comment]
Pufferfish added

**Comment by Tofpu** [comment]
> Would you mind ticking the ones off we have done? Done: Purpur/Paper/Spigot/Bukkit/Vanilla

If you would like to cross them out yourself, I think you might be able to use the edit button on Dawsson's message to do it. Of course, this will depend on your privileges on the organization and/or repository and whatnot
![image](https://github.com/flytegg/mcu-website/assets/47629321/e9d821d4-ad31-4924-a5cb-e6a572970894)

**Comment by Dawsson** [comment]
I could just like edit the message yk

**Comment by joshbker** [comment]
Smh @Dawsson use Markdown !!

- [ ] Purpur
- [x] Purpur
```
- [ ] Purpur
- [x] Purpur
```

**Comment by Dawsson** [comment]
ohg just liek my obsidian vault!

**Comment by joshbker** [comment]
> ohg just liek my obsidian vault!

yes and without clicking edit you can click on the boxes to tick them off (so can me, stephen, and others with perms). you can also drag them around to rearrange the order

**Comment by Pan-nav** [comment]
Is this the same as https://github.com/flytegg/mcu-website/issues/148 ? If so you can close one of them?

**Comment by Dawsson** [comment]
Closed, ty

**Comment by sinnayuh** [comment]
Some more you could potentionally add

- NeoForge
- Quilt
